dot/peerset/peerset.go

```diff
@@ -380,6 +384,7 @@ func (ps *PeerSet) allocSlots(setIdx int) error {
 		}
 
 		if err = peerState.tryOutgoing(setIdx, peerID); err != nil {
+			logger.Errorf("could not set peer as outgoing connection, peer: %s, error: %s", peerID.Pretty(), err)
```

Comment:
nit make it short and human readable (aka left to right)

```suggestion
			logger.Errorf("could not set peer %s as outgoing connection: %s", peerID.Pretty(), err)
```